### DISCUSSION TITLE
담당 주제 선택

### DISCUSSION THREAD

**Discussion by AlbertImKr** [weaverse-techtide/weaverse-backend#37] [Ideas]
-  담당자: 유정님
    - #10 
    - #15 
    - #16 

- 담당자: 원길님
	- #14
- 담당자: 홍광님
	- #21 
	- #17 
## 2 주차
- 담당자
	- #22

**Comment by devnproyj22** [comment]
1주차 인증, 미션 기능 구현 희망합니다.

**Comment by nathanLYJ** [comment]
1 주차 인증 기능 + 사용자 관리 기능 희망합니다.
